src/styles/_variables.scss

```diff
@@ -1,3 +1,6 @@
+// @import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100;300;400;500;700;900&display=swap');
```

Comment:
it's a good habit to get into removing commented code
(I may have left some myself when initializing this repo - but intend to go back and delete the commented code)